models/__init__.py

```diff
@@ -1,13 +1,26 @@
 # coding=utf-8
+import pandas as pd
 from sqlalchemy import func
-from models.cyberconnect2 import Session
+from models.cyberconnect2 import Session, engine
+
 from models.t_ice_time import Ice_Time
 from models.t_locations import Locations
 from models.t_icetype import IceType
+from models.t_coaches import Coaches
+
 
 session = Session()
 
 
+class coaches():
+
+    def add_coaches(coaches):
+        for coach in coaches:
+            session.add(Coaches(**coach))
+            session.commit()
+        session.close()
```

Comment:
A session is prematurely and unexpectedly closed here. Consider utilizing context-manager(`with Session(...) as session`), or pass the session (db connection) to the class/function.